squad/api/rest.py

```diff
@@ -833,6 +833,23 @@ class AnnotationViewSet(viewsets.ModelViewSet):
     ordering_fields = ('id', 'build')
 
 
+class MetricThresholdSerializer(serializers.ModelSerializer):
+
+    id = serializers.IntegerField(read_only=True)
+
+    class Meta:
+        model = MetricThreshold
+        fields = '__all__'
+
+
+class MetricThresholdViewSet(viewsets.ModelViewSet):
+
+    queryset = MetricThreshold.objects.all()
+    serializer_class = MetricThresholdSerializer
+    filter_fields = ('name', 'value', 'is_higher_better', 'project')
```

Comment:
Please add MetricThresholdFilter and add filter_class here. There is at least one related field here (project) that can be traversed.